tpl/tplimpl/embedded/templates/shortcodes/vimeo_simple.html

```diff
@@ -0,0 +1,19 @@
+{{ $id := .Get "id" | default (.Get 0) }}
+{{- $item := getJSON "https://vimeo.com/api/oembed.json?url=https://vimeo.com/" $id -}}
+{{ $class := .Get "class" | default (.Get 1) }}
+{{ $hasClass := $class }}
+{{ $class := $class | default "__h_video" }}
+{{ if not $hasClass }}
+{{/* If class is set, assume the user wants to provide his own styles. */}}
+{{ template "__h_simple_css" $ }}
+{{ end }}
+{{ $secondClass := "s_video_simple" }}
+<div class="{{ $secondClass }} {{ $class }}">
+{{- with $item }}
+<a href="{{ .provider_url }}{{ .video_id | safeHTMLAttr }}" target="_blank">
+{{/* (onedrawingperday) Vimeo API adds an underscore and dimensions in the file name of low res thumbnails. So removing these with a bit of RegEx will give us the high resolution for use in a srcset*/}}
```

Comment:
Comments are good, "named comments" does not make much sense. I tend to mark TODOs with my name (I see other project do that), to get a reminder ("I put that there") when I come back to it later.

Reply:
Other than those minor comments, this PR looks smashing.

Reply:
This one also. I just wanted you to know what I did.